Dockerfile

```diff
@@ -1,5 +1,16 @@
-FROM adoptopenjdk/openjdk11:alpine-jre
-ADD https://github.com/microsoft/ApplicationInsights-Java/releases/download/3.2.6/applicationinsights-agent-3.2.6.jar /applicationinsights-agent.jar
+FROM maven:3.8.4-jdk-11-slim as buildtime
+
+WORKDIR /build
+COPY . .
+
+RUN mvn clean package
```

Comment:
@alberto-re-pagopa Using a multistage build may be a cleaner solution, but we're already building the project in the deploy pipeline. As a result, we're build it two times